spec/models/emote_spec.rb

```diff
@@ -1,18 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Emote do
-    let(:emotion) { Emotion.new(name: "joy") }
-    let(:action) { Action.new(content: "Mow your lawn (joy)") }
-    let(:recommendation) { Recommendation.new(emotion: emotion, action: action) }
-    let(:emote) { Emote.new(input: "I am happy", recommendation: recommendation) }
+  let(:emotion) { Emotion.new(name: "joy") }
+  let(:user) { User.new(identifier: "+1555-555-1234", identifier_type: "sms") }
+  let(:emote) { Emote.new(emote: "I am happy", user: user, emotion: emotion) }
 
-    it "has user input" do 
-        expect(emote.input).to eq "I am happy"
-    end
+  it "has user input" do
+    expect(emote.emote).to eq "I am happy"
+  end
 
-    it "has a recommendation" do 
-        expect(emote.recommendation).to be recommendation
-    end
+  it "is associated with a user" do
+    expect(emote.user).to be user
+  end
 
-    # Not testing retrieve_action to avoid using up our allotted api calls
-end
\ No newline at end of file
+  it "has an emotion associated with it" do
+    expect(emote.emotion).to be emotion
```

Comment:
I would reword this so it's not "self-fulfilling" - instead of `.to be emotion`, maybe `.to be "joy"`